### PR TITLE
#87 adding more tests with mocking

### PR DESCRIPTION
**Pull request by aanersc**

### CODE REVIEW

File: geospaas/nansat_ingestor/tests/tests.py [hunk 1]
```diff
@@ -87,19 +88,45 @@ def mock_bands(self):
...
         mock_isfile.return_value = True
         uri = 'file://localhost/some/folder/filename.ext'
         ds0, cr0 = Dataset.objects.get_or_create(uri)
         ds1, cr1 = Dataset.objects.get_or_create(uri)
 
         self.assertTrue(cr0)
         self.assertFalse(cr1)
+
+    @patch('os.path.isfile')
+    def test_getorcreate_localfile_is_mached_in_metadata(self, mock_isfile):
```
Comment (akorosov) [inline geospaas/nansat_ingestor/tests/tests.py:101]:
```suggestion
    def test_getorcreate_localfile_is_matched_in_metadata(self, mock_isfile):
```

File: geospaas/nansat_ingestor/tests/tests.py [hunk 2]
```diff
@@ -87,19 +88,45 @@ def mock_bands(self):
...
+        ds1, cr1 = Dataset.objects.get_or_create(uri)
+        self.assertEqual(ds0.parameters.values()[
+                         0]['short_name'], self.predefined_band_metadata_dict[2]['short_name'])
+
+    @patch('os.path.isfile')
+    def test_getorcreate_localfile_matched_parameter(self, mock_isfile):
+        mock_isfile.return_value = True
+        uri = 'file://localhost/some/folder/filename.ext'
+        ds0, cr0 = Dataset.objects.get_or_create(uri)
+        ds0.save()
```
Comment (akorosov) [inline geospaas/nansat_ingestor/tests/tests.py:125]:
```suggestion
```

File: geospaas/nansat_ingestor/tests/tests.py [hunk 3]
```diff
@@ -87,19 +88,45 @@ def mock_bands(self):
...
+    def test_getorcreate_localfile_matched_parameter(self, mock_isfile):
+        mock_isfile.return_value = True
+        uri = 'file://localhost/some/folder/filename.ext'
+        ds0, cr0 = Dataset.objects.get_or_create(uri)
+        ds1, cr1 = Dataset.objects.get_or_create(uri)
+        self.assertEqual(ds0.parameters.values()[
+                         0]['short_name'], self.predefined_band_metadata_dict[2]['short_name'])
+
+    @patch('os.path.isfile')
+    def test_getorcreate_localfile_matched_parameter(self, mock_isfile):
```
Comment (akorosov) [inline geospaas/nansat_ingestor/tests/tests.py:121]: Rename method

**Comment by aanersc** [comment]
@akorosov 
Dear anton,
New lines of code are added to fd09828 and the writing aspect of the files are considered in b760713. There is also a tiny useless comment removal in the dd4d4eb. Please find them all.
The final one is dd4d4eb.

**Pull request merged**